### DISCUSSION TITLE
TS nets/whisper to dynamically change based on a root channel you're sat in...

### DISCUSSION THREAD

**Discussion by spamenigma** [skyeBreach/ueepathfinders-helpdesk#4] [Ideas]
### Describe the feature you'd like to request

With multiple ships with a similar structure (bridge/gunnery/security/medical/cargo.. etc..etc..)

If I'm on the bridge of Ship A, I may need to use a net to talk to Security on Ship A 

If I'm on ship B, I want the same key bind/button to talk to Security.. but that will be Security of Ship B>

This allows people to move between ships and not have to mess with whisper lists/keybinds

### Describe the solution you'd like to see

.

**Comment by skyeBreach** [comment]
After speaking with spam we both agreed that an impromptu ship "intercom" system would be an ideal implementation. Where instead of/or aswell as the hotkey whisper system a UI element/window could be used to choose the section of the ship (channel) you want to intercom too. 

Using a UI element/window would remove the current issue where most pathfinders do not have enough keys available to set all whisper lists/nets. This same idea can be expanded to the rest of the nets, as a back up to TeamSpeaks standard keybinding system.
